README.md

```diff
@@ -38,14 +38,21 @@ can be handled.
 
 It has been tested with Ruby 2.6.
 
+### Usage / Updating
+
+```fish
+./gen_completions.rb docker > completions/docker.fish
+./gen_completions.rb docker-compose > completions/docker-compose.fish
+```
+
 ## License
 
 [MIT](LICENCE.md)
 
-[Docker]: https://www.docker.com
+[docker]: https://www.docker.com
```

Comment:
I think the case has to match, doesn’t it?

Reply:
Nope, case-normalized (https://github.github.com/gfm/#matches)

Reply:
Mmm. I’m going to end up overriding that, because as far as I know, the original Markdown spec indicates case sensitivity (and I think that MMD does as well). GFM is an implementation of Markdown, but it isn’t the only implementation. Not a big deal.